sentry/src/main/java/io/sentry/Dsn.java

```diff
@@ -49,6 +49,9 @@ URI getSentryUri() {
 
   Dsn(@Nullable String dsn) throws IllegalArgumentException {
     try {
+      if (dsn == null) {
+        throw new IllegalArgumentException("Invalid DSN: Cannot be null.");
+      }
```

Comment:
```suggestion
    Objects.requireNonNull(dsn, "The DSN is required.");
```
to be consistent with our other null checks